tests/console/upgrade_snapshots.pm

```diff
@@ -23,12 +23,15 @@ sub run {
     select_console 'root-console';
 
     my $waittime = 200 * get_var('TIMEOUT_SCALE', 1);
-    script_run("snapper list | tee /dev/$serialdev", 0);
+    my $ret      = script_run("snapper --help | grep disable-used-space");
```

Comment:
If I am not mistaken the script_run call will cause a red failure box if returning false which can confuse test reviewers.

I suggest to prevent the switch between local bash and remote Perl code. How about the following:

```perl
script_run("(snapper --help | grep -q disabled users space && snapper list --disable-used-space || snapper list) | tee /dev/$serialdev", 0);
```

Also I suggest a comment in code to explain the reason for the additional option

Reply:
I've changed the code with your suggestion and would you please take a look again? thanks!

Reply:
yes, looks good.